en/program.html

```diff
@@ -188,5 +188,11 @@ <h3 class="Text--little Header Header--third">Lightning Talk</h3>
       </tbody>
     </table>
   </div><!-- /.FutureRegion -->
+  <p>Do you think #pyconapac2015 Activities of second day at the end of 5:30 pm on June 6?
+No! #pyconapac Wonderful dinner is yet to come! Not only delicious buffet with a real treat for you,
+There are many eye-opening performance turns on stage. If this is not enough excitement,
+It's okay! The stall also invited everyone to come and work with us to create their PyCon bustling night party! </p>
```

Comment:
"If this is not enough excitement, It's okay! ..." I might do some rewording here:

- It's "our" PyCon night party :)
- In /program/night-party, stall here is referred to booth. I will unify the term using "booth"
- In the Chinese version, I think this message is more about inviting them to apply for booth in the night party. My modification will target for the same idea

All in all I think you did a wonderful job! That's the most touching passage I've seen in our site :+1: